DataManagementSystem/Service/StorageElementHandler.py

```diff
@@ -102,16 +102,16 @@ def getFreeDiskSpace():
   global BASE_PATH
 
   result = getDiskSpace(BASE_PATH)  # free
-  if not result['OK']:
+  if not MAX_STORAGE_SIZE or not result['OK']:
```

Comment:
This is OK but I would split it in two if's because the reasons to exit here are completely different and the code takes time to guess the logic

Reply:
```suggestion
  if not result['OK']:
    return result
   if not MAX_STORAGE_SIZE:
     return result
```
like this?

Reply:
Perfect :)